src/client/package.json

```diff
@@ -21,15 +21,16 @@
   },
   "scripts": {
     "test": "jest",
-    "lint": "eslint",
+    "lint": "eslint .",
+    "lint-staged": "lint-staged",
     "start": "cross-env TARGET=web vite",
     "build": "cross-env TARGET=web vite build",
     "serve": "cross-env TARGET=web vite preview",
     "openfin:dev": "cross-env TARGET=openfin vite",
     "_openfin:run": "cross-env-shell \"wait-on -l $npm_config_manifest_url && openfin -l -c $npm_config_manifest_url\"",
-    "openfin:run:fx": "npm run _openfin:run --manifest_url=http://localhost:1917/dist/config/rt-fx.json",
-    "openfin:run:credit": "npm run _openfin:run --manifest_url=http://localhost:1917/dist/config/rt-credit.json",
-    "openfin:run:launcher": "npm run _openfin:run --manifest_url=http://localhost:1917/dist/config/launcher.json",
+    "openfin:run:fx": "npm run _openfin:run --manifest_url=http://localhost:1917/config/rt-fx.json",
+    "openfin:run:credit": "npm run _openfin:run --manifest_url=http://localhost:1917/config/rt-credit.json",
+    "openfin:run:launcher": "npm run _openfin:run --manifest_url=http://localhost:1917/config/launcher.json",
```

Comment:
perhaps we can remove `openfin:run:preview` as it is now identical to `openfin:run:launcher`; we can update `openfin:start:preview` to use `openfin:run:launcher` instead

Reply:
good point .. that script does not make sense now .. the start:preview is probably still ok, but call the openfin:run:launcher instead to kick off openfin after build/serve